src/components/TodoFilters/TodoFilters.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+export const TodoFilters = ({ filterTodosByStatus, activeFilter }) => (
+  <ul className="filters">
+    <li>
+      <a
+        href="#/"
+        className={classnames({
+          selected: activeFilter === 'all',
+        })}
+        onClick={() => {
+          filterTodosByStatus('all');
+        }}
+      >
+        All
+      </a>
+    </li>
+
+    <li>
+      <a
+        href="#/active"
+        className={classnames({
+          selected: activeFilter === 'active',
+        })}
+        onClick={() => {
+          filterTodosByStatus('active');
+        }}
+
+      >
+        Active
+      </a>
+    </li>
+
+    <li>
+      <a
+        href="#/completed"
+        className={classnames({
+          selected: activeFilter === 'completed',
```

Comment:
let's create a constant for all filter types, so you can import them everywhere and use like this (FILTERS.all, etc)